petastorm/tests/test_shuffling_buffer.py

```diff
@@ -203,8 +301,10 @@ def test_noop_shuffling_buffer_stream_through(buffer_type):
 def test_batched_random_shuffling_buffer_stream_through(batch_size):
     """Feed a 0:99 sequence through a BatchedRandomShufflingBuffer. Check that the order has changed."""
     input_sequence = range(100)
-    a = _feed_a_sequence_through_the_queue(BatchedRandomShufflingBuffer(10, 3, batch_size), input_sequence)
-    b = _feed_a_sequence_through_the_queue(BatchedRandomShufflingBuffer(10, 3, batch_size), input_sequence)
+    a = _feed_a_sequence_through_the_queue(
```

Comment:
In the case of batch_size of 10, `_feed_a_sequence_through_the_queue` falls into a forever loop where it can neither add or retrieve samples from the buffer. It is not a but, simply, those numbers are a weird combination.